scripts/build_upload.py

```diff
@@ -0,0 +1,476 @@
+    print()
+    banner(red, "{:^80}".format("Bokeh %r build and upload: FAILURE" % CONFIG.version))
+    sys.exit(1)
+
+def abort_uploads():
+    print(red("\n!!! FATAL problems occurred during UPLOADS"))
+    print()
+    print(bright(red("!!! SOME ASSETS MAY HAVE BEEN UPLOADED")))
+    print()
+    print(bright(yellow("Here is the status of all uploads:")))
+    print()
+    for upload in CONFIG.uploads:
+        print("    - %-10s: %s" % (upload, CONFIG.upload_status[upload]))
+    print()
+    banner(red, "{:^80}".format("Bokeh %r build and upload: FAILURE" % CONFIG.version))
+    sys.exit(1)
+
+#--------------------------------------
+#
+# Check functions
+#
+#--------------------------------------
+
+def check_environment_var(name, message):
+    if name in os.environ:
+        passed("Found %s (%s)" % (message, name))
+    else:
+        failed("Missing %s (%s)" % (message, name))
+        abort_checks()
+
+def check_anaconda_creds():
+    try:
+        token = os.environ['ANACONDA_TOKEN']
+        out = run("anaconda -t %s whoami" % token, silent=True)
+        if "Anonymous User" in out:
+            failed("Could NOT verify Anaconda credentials")
+            abort_checks()
+        passed("Verified Anaconda credentials")
+    except Exception as e:
+        failed("Could NOT verify Anaconda credentials")
+        abort_checks()
+
+def check_cdn_creds():
+    try:
+        username = os.environ['RSUSER']
+        key = os.environ['RSAPIKEY']
+        buf = BytesIO()
+        c = pycurl.Curl()
+        c.setopt(c.CAINFO, certifi.where())
+        c.setopt(c.URL, 'https://identity.api.rackspacecloud.com/v2.0/tokens/')
+        c.setopt(c.HTTPHEADER, ['Content-type: application/json'])
+        c.setopt(c.POSTFIELDS, json.dumps({
+            "auth" : {
+                "RAX-KSKEY:apiKeyCredentials" : {
+                    "username" : username,
+                    "apiKey"   : key,
+                }
+            }
+        }))
+        c.setopt(c.WRITEDATA, buf)
+        c.perform()
+        c.close()
+
+        data = json.loads(buf.getvalue().decode())
+
+        cdn_token = data["access"]["token"]["id"]
+        cdn_id = [
+            d["endpoints"][0]["tenantId"]
+            for d in data["access"]["serviceCatalog"]
+            if d["name"] == "cloudFiles"
+        ][0]
+
+        passed("Retrieved CDN credentials")
+        return cdn_token, cdn_id
+    except:
+        failed("Could NOT retrieve CDN credentials")
+        abort_checks()
+
+#--------------------------------------
+#
+# Build functions
+#
+#--------------------------------------
+
+@build_wrapper('conda')
+def build_conda_packages():
+    for v in "27 34 35 36".split():
+        os.environ['CONDA_PY'] = v
+        run("conda build conda.recipe --quiet")
+        # TODO (bev) make platform detected or configurable
+    files = " ".join(glob.glob('/home/travis/miniconda/conda-bld/linux-64/bokeh*'))
+    run("conda convert -p all %s" % files)
```

Comment:
>But it would be irresponsible to switch to them before it can be reasonably expected that most users have updated to a conda that supports them.

OK, I agree.